src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -463,29 +428,16 @@ void BaseCouplingScheme::checkCompletenessRequiredActions()
   }
 }
 
-void BaseCouplingScheme::setupDataMatrices(DataMap &data)
+void BaseCouplingScheme::setupDataMatrices()
 {
   PRECICE_TRACE();
-  PRECICE_DEBUG("Data size: {}", data.size());
-  // Reserve storage for convergence measurement of send and receive data values
-  for (ConvergenceMeasureContext &convMeasure : _convergenceMeasures) {
-    PRECICE_ASSERT(convMeasure.couplingData != nullptr);
-    if (convMeasure.couplingData->oldValues.cols() < 1) {
-      utils::append(convMeasure.couplingData->oldValues,
-                    (Eigen::MatrixXd) Eigen::MatrixXd::Zero(convMeasure.couplingData->values().size(), 1));
-    }
+  // Reserve storage for all data (acceleration and convergence measurement)
```

Comment:
```suggestion
  // Reserve storage for all data
```
We do it for all data now, right? Not only such used in acc or cm.